2022-07-10 19:37:54,384 INFO    MainThread:24505 [wandb_setup.py:_flush():76] Configure stats pid to 24505
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_setup.py:_flush():76] Loading settings from /home/will/.config/wandb/settings
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_setup.py:_flush():76] Loading settings from /home/will/projects/RL/RLBoard/wandb/settings
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'viz_hot_starts_SAC.py', 'program': 'viz_hot_starts_SAC.py'}
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_init.py:_log_setup():461] Logging user logs to /home/will/projects/RL/RLBoard/wandb/run-20220710_193754-1nme6emd/logs/debug.log
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_init.py:_log_setup():462] Logging internal logs to /home/will/projects/RL/RLBoard/wandb/run-20220710_193754-1nme6emd/logs/debug-internal.log
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_init.py:init():495] calling init triggers
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_init.py:init():499] wandb.init called with sweep_config: {}
config: {}
2022-07-10 19:37:54,385 INFO    MainThread:24505 [wandb_init.py:init():548] starting backend
2022-07-10 19:37:54,385 INFO    MainThread:24505 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-07-10 19:37:54,390 INFO    MainThread:24505 [backend.py:ensure_launched():217] starting backend process...
2022-07-10 19:37:54,394 INFO    MainThread:24505 [backend.py:ensure_launched():222] started backend process with pid: 24524
2022-07-10 19:37:54,396 INFO    MainThread:24505 [wandb_init.py:init():558] backend started and connected
2022-07-10 19:37:54,398 INFO    MainThread:24505 [wandb_init.py:init():634] updated telemetry
2022-07-10 19:37:54,406 INFO    MainThread:24505 [wandb_init.py:init():666] communicating run to backend with 30 second timeout
2022-07-10 19:37:55,049 INFO    MainThread:24505 [wandb_run.py:_on_init():1919] communicating current version
2022-07-10 19:37:55,197 INFO    MainThread:24505 [wandb_run.py:_on_init():1923] got version response 
2022-07-10 19:37:55,197 INFO    MainThread:24505 [wandb_init.py:init():700] starting run threads in backend
2022-07-10 19:37:57,839 INFO    MainThread:24505 [wandb_run.py:_console_start():1893] atexit reg
2022-07-10 19:37:57,840 INFO    MainThread:24505 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-07-10 19:37:57,840 INFO    MainThread:24505 [wandb_run.py:_redirect():1771] Redirecting console.
2022-07-10 19:37:57,842 INFO    MainThread:24505 [wandb_run.py:_redirect():1827] Redirects installed.
2022-07-10 19:37:57,843 INFO    MainThread:24505 [wandb_init.py:init():732] run started, returning control to user process
2022-07-10 19:38:01,044 INFO    MainThread:24505 [wandb_watch.py:watch():47] Watching
2022-07-10 19:39:28,240 ERROR   MainThread:24505 [util.py:get_module():305] Error importing optional module moviepy.editor
Traceback (most recent call last):
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/wandb/util.py", line 300, in get_module
    return import_module(name)
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/editor.py", line 36, in <module>
    from .video.io.VideoFileClip import VideoFileClip
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/video/io/VideoFileClip.py", line 3, in <module>
    from moviepy.audio.io.AudioFileClip import AudioFileClip
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/audio/io/AudioFileClip.py", line 3, in <module>
    from moviepy.audio.AudioClip import AudioClip
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/audio/AudioClip.py", line 7, in <module>
    from moviepy.audio.io.ffmpeg_audiowriter import ffmpeg_audiowrite
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/audio/io/ffmpeg_audiowriter.py", line 7, in <module>
    from moviepy.config import get_setting
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/moviepy/config.py", line 36, in <module>
    FFMPEG_BINARY = get_exe()
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/imageio/plugins/ffmpeg.py", line 51, in get_exe
    return imageio_ffmpeg.get_ffmpeg_exe()
  File "/home/will/anaconda3/envs/mujoco/lib/python3.7/site-packages/imageio_ffmpeg/_utils.py", line 50, in get_ffmpeg_exe
    "No ffmpeg exe could be found. Install ffmpeg on your system, "
RuntimeError: No ffmpeg exe could be found. Install ffmpeg on your system, or set the IMAGEIO_FFMPEG_EXE environment variable.
2022-07-10 19:39:28,243 INFO    MainThread:24505 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-07-10 19:39:28,245 INFO    MainThread:24505 [wandb_run.py:_restore():1834] restore
2022-07-10 19:39:31,214 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 817
  total_bytes: 817
}

2022-07-10 19:39:31,377 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 817
  total_bytes: 817
}

2022-07-10 19:39:32,111 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 817
  total_bytes: 10931
}

2022-07-10 19:39:32,218 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 817
  total_bytes: 10958
}

2022-07-10 19:39:32,321 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:32,424 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:32,526 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:32,628 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:32,730 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:33,046 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}

2022-07-10 19:39:33,336 INFO    MainThread:24505 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10958
  total_bytes: 10958
}
local_info {
}

2022-07-10 19:39:34,470 INFO    MainThread:24505 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-07-10 19:39:34,470 INFO    MainThread:24505 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-07-10 19:39:34,472 INFO    MainThread:24505 [wandb_run.py:_footer_sync_info():3055] logging synced files
